Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -503,14 +527,29 @@ class BocList
     {
       if (!PageUtility.Instance.IsInDom (scrollableContainer))
         return;
-  
+
+      BocList.FixValidationErrorOverflowSize(scrollableContainer);
       BocList.FixHeaderSize(scrollableContainer);
       BocList.FixHeaderPosition(tableContainer, scrollableContainer);
       setTimeout(resizeHandler, resizeInterval);
     };
     resizeHandler();
   }
 
+  private static FixValidationErrorOverflowSize(scrollableContainer: HTMLElement)
+  {
+    // Sets the width of the validation error field to be as wide as the visible part of the bocList.
+    // This enables sticky behaviour while still allowing for a scrollbar if required.
+
+    const width = LayoutUtility.GetWidth(scrollableContainer) - 1;
+
+    const overflowContainer: Nullable<HTMLElement> = scrollableContainer.querySelector('div.bocListTableBlock tr.bocListValidationRow div');
+    if (overflowContainer)
+    {
+      //overflowContainer.style.width = width.toString() + "px";
```

Comment:
that's commented out?

Reply:
Yeah, my comment above is about this: Due to the colspan rendering, we now cannot just apply the width of the visible part of the bocList, as this would cause overflowing. As this still needs fixing, I just commented it out.

Reply:
Unless I have no clue how this works and it's really easy, I don't think this should happen in this task.

This will probably take significantly more time than I think would be reasonable to append onto this PR now (0.5 - 1pt I would guess, unless there's a really easy way to do it...which I do not know)